modules/dxp/apps/commerce/commerce-shop-by-diagram/commerce-shop-by-diagram-web/src/main/resources/META-INF/resources/style/icons.svg

```diff
@@ -0,0 +1,1914 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE svg PUBLIC "-//W3C//DTD SVG 1.1//EN" "http://www.w3.org/Graphics/SVG/1.1/DTD/svg11.dtd">
+<svg xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink">
```

Comment:
Are these icons intended to be included on Clay? /cc @matuzalemsteles

Reply:
Wow, I imagine this is a copy of our icons, in theory, you don't need to add this file because we already do that, you can get the link through `window.Liferay.ThemeDisplay.getPathThemeImages() + '/clay/icons.svg'` if you are not using `render`.

Reply:
Just to clarify, i added a spritemap icon only to make the component work also in a static env. Am i wrong?

Reply:
@garofalog if this is meant to be used outside a Liferay environment, no problem, but as this now goes inside the Liferay environment, in that case, you don't need to upload these icons together unless you've added other icons that aren't around the pattern being part of the Lexicon.